src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -22,7 +22,13 @@ import { ChromeNavGroupServiceStartContract, NavGroupItemInMap } from '../../nav
 import { ChromeNavLink } from '../../nav_links';
 import { ALL_USE_CASE_ID } from '../../../../../core/utils';
 import { fulfillRegistrationLinksToChromeNavLinks } from '../../utils';
-import './collapsible_nav_group_enabled_top.scss';
+
+// Reference https://github.com/opensearch-project/oui/blob/main/src/services/color/is_valid_hex.ts
+const validateWorkspaceColor = (color?: string) =>
+  !!color && /(^#[0-9A-F]{6}$)|(^#[0-9A-F]{3}$)/i.test(color);
```

Comment:
This was already defined in src/plugins/workspace/common/utils.ts. We should move it to the @osd/std package and reuse it in both places.

That said, a better way of checking the validity of the color is
```js
const isColorValid(color: string) {
  const { style } = new Option();
  style.color = color;
  return !['', 'unset', 'inherit', 'initial'].includes(style.color);
}
```

This will accept hex and string colors, as well as more complex ones.

Reply:
it seems we do not need to verify if the color is valid, it need to be verified in workspace page because it will pass the color into color-picker component. But thanks anyway.